resources/views/torrent/partials/buttons.blade.php

```diff
@@ -393,9 +393,10 @@ class="form__button form__button--outlined"
         <button
             class="form__button form__button--outlined form__button--centered"
             x-bind="showDialog"
+            title="This torrent currently has {{ $torrent->reports->where('solved', '=', 0)->count() }} unsolved report(s)"
```

Comment:
Can this be moved into the torrents controller? Also, a relation in models/Torrent.php would be good as well.

Reply:
Done.